config/routes.rb

```diff
@@ -34,11 +34,10 @@
   get  '/cant-parse', to: 'runs#cant_parse', as: :cant_parse
   get  '/random',     to: 'runs#random',     as: :random
 
-  get  '/convert',    to: 'converts#new'
+  get  '/convert', to: 'converts#new'
 
-  get '/search',        to: 'search#index'
-  get '/search(?q=:q)', to: 'search#index'
```

Comment:
I'm not super great with routing, is this route not needed when the base route without the query is defined?

Reply:
Nope it is not. A request to `GET /search?q=mario` will still match the `get '/search'` route and `params[:q]` will be set to `"mario"`.

Reply:
Good to know, thanks for the info.